11/hmw-11-01.py

```diff
@@ -0,0 +1,20 @@
+# Написати власний декоратор, задачею якого має бути друк назви функції і часу, коли вона була викликана.
+# Декоратор має працювати для різних функцій однаково.
+
+from datetime import datetime
+
+
+def my_decorator(func):
+    def deco_func():
```

Comment:
`deco_func` should take *args and **kwargs as parameters and then you should pass these parameters as arguments to your `func` call. This will allow decorating functions that take parameters